### ISSUE TITLE
Read worksheet throw exception

### ISSUE THREAD

**Issue by H0LIXI** [EPPlusSoftware/EPPlus#1528]
### EPPlus usage

Noncommercial use

### Environment

Windows

### Epplus version

7.1.2

### Description

Some validation data will cause exception
I have provided a test project, which contains a test.xlsx.
[TestEPPlusReadSheet.zip](https://github.com/user-attachments/files/16279307/TestEPPlusReadSheet.zip)

My code
```
using (var excel = new ExcelPackage(excelPath))
{
    var worksheets = excel.Workbook.Worksheets;
    foreach (var worksheet in worksheets)
    {
        Console.WriteLine($"Read Sheet: {worksheet.Name}. Rows[{worksheet.Dimension?.Rows ?? 0}] Cols[{worksheet.Dimension?.Columns ?? 0}]");
    }
}
```

Exception
```
System.FormatException: String was not recognized as a valid Boolean.
   at System.Boolean.Parse(String value)
   at OfficeOpenXml.DataValidation.ExcelDataValidationList.LoadXML(XmlReader xr)
   at OfficeOpenXml.DataValidation.ExcelDataValidationFactory.Create(XmlReader xr, ExcelWorksheet ws)
   at OfficeOpenXml.DataValidation.ExcelDataValidationCollection.ReadDataValidations(XmlReader xr)
   at OfficeOpenXml.ExcelWorksheet.CreateXml()
   at OfficeOpenXml.ExcelWorksheet..ctor(XmlNamespaceManager ns, ExcelPackage excelPackage, String relID, Uri uriWorksheet, String sheetName, Int32 sheetID, Int32 positionID, Nullable`1 hide)
   at OfficeOpenXml.ExcelWorksheets..ctor(ExcelPackage pck, XmlNamespaceManager nsm, XmlNode topNode)
   at OfficeOpenXml.ExcelWorkbook.get_Worksheets()
```

**Comment by swmal** [comment]
@H0LIXI - we will have a look at this.

**Comment by OssianEPPlus** [comment]
This is a bug that was resolved in version 7.1.3 the most current version is 7.2.1 not 7.1.2
Update the epplus package and it should work.
